packages/css/src/components/button/button.scss

```diff
@@ -28,7 +28,7 @@
 
 @mixin ams-button-forced-color-mode {
   @media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
-    border: 2px solid ButtonBorder; // add border because forced colors changes box-shadow to none
+    border: var(--ams-button-forced-color-mode-border); // add border because forced colors changes box-shadow to none
```

Comment:
Should we add a ticket to check all our box shadows for forced color mode and add some docs for it?

Reply:
Yes, separating feels better because it doesn’t mix up testing with adding features.